packages/react/fast-components-react-base/src/data-grid/data-grid.schema.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Complies with FAST Tooling 1.0
```

Comment:
looks like there are no react children, you can consolidate these and just use one schema since there are no differences.